docs/05-Navigation Interception.md

```diff
@@ -0,0 +1,16 @@
+## Navigation Interception
+
+By default, PalindromDOM intercepts browser navigation and sends an AJAX request to the server for the next page's partial/model. This results in a more efficient and faster navigation.
```

Comment:
The word "partial" does not exist in Palindrom docs by now. For a very good reason: Palindrom is not only meant for Starcounter views, but in general, for "two-way data binding between local and remote JSON models".

I propose to rewrite this sentence as following:

```
PalindromDOM gives a regular HTML document the benefits of a single-page application 
by adding a special optimisation to the internal web links. 

Regular links in the current host are cancelled and replaced by a GET  `XmlHttpRequest` to 
the same URL with `Accept: application/json-patch+json`. Upon this request, the server 
modifies the model and responds with a patch that contain the minimum set of changes 
to navigate to the URL.

The browser navigation (i.e. the back, forward and reload buttons) is preserved by the use 
of the HTML History API.

Only links in the current host without the target attribute or with the target attribute set 
to `_self` are intercepted.
```